Examples/Python/python/acts/examples/odd.py

```diff
@@ -99,6 +100,22 @@ def geoid_hook(geoid, surface):
             level=customLogLevel(minLevel=acts.logging.WARNING),
         )
 
-    trackingGeometry, deco = detector.finalize(dd4hepConfig, mdecorator)
+    trackingGeometry, decorators = detector.finalize(dd4hepConfig, mdecorator)
 
-    return detector, trackingGeometry, deco
+    OpenDataDetector = namedtuple(
+        "OpenDataDetector", ["detector", "trackingGeometry", "decorators"]
+    )
+
+    class OpenDataDetectorContextManager(OpenDataDetector):
```

Comment:
Dies this duplicate the automatically attached wrapper?

Reply:
The code is mostly duplicated but we should only end up with a single context manager since we call `finalize` manually here. I didn't know how to best deduplicate this but it also has a different behavior so I thought it might be fine to have these few lines 2 times